playbooks/container/docker-rootless.yml

```diff
@@ -0,0 +1,7 @@
+---
+- hosts: all
+  become: yes
+  roles:
+    - name: docker-rootless
+      vars:
+        rootlessdocker_install_dir: "{{ sm_software_path }}/rootless-docker"
```

Comment:
This should be removed since it exists as a default in the role. Our convention is also to add any default variables to the appropriate `config.example/group_vars` files.

Reply:
Will do.